### PR TITLE
Add documentation about filter operators

### PR DESCRIPTION
**Pull request by fzaninotto**
Adding this doc show that #7122 isn't required.

### CODE REVIEW

File: docs/FilteringTutorial.md [hunk 1]
```diff
@@ -171,6 +171,62 @@ If you use the FilterList, you'll probably need a search input. As the FilterLis
...
+The demos show one possible way: suffix the filter name with an operator, e.g. "_gte" for "greater than or equal to".
+
+```jsx
+const postFilters = [
+    <DateInput source="released_gte" label="Released after" />
+    <DateInput source="released_lte" label="Released before" />
+];
+```
+
+Some API backend (e.g. JSON Server) know how to handle this syntax. If your API doesn't understand these 'virtual fields', you will have to transform them into the expected syntax in the Data Provider.
```
Comment (djhi) [inline docs/FilteringTutorial.md:195]:
```suggestion
Some API backends (e.g. JSON Server) know how to handle this syntax. If your API doesn't understand these 'virtual fields', you will have to transform them into the expected syntax in the Data Provider.
```

**Review by djhi — CHANGES_REQUESTED** [review]

File: docs/FilteringTutorial.md [hunk 2]
```diff
@@ -171,6 +171,62 @@ If you use the FilterList, you'll probably need a search input. As the FilterLis
...
+
+This is fine for equality filters, but how can you do more complex filters, like "between", "contains", "starts with", "greater than", etc.?
+
+As there is no standard way to pass such complex filters to APIs, react-admin makes no decision about it. It's up to you to decide how to store them in the filter object.
+
+The demos show one possible way: suffix the filter name with an operator, e.g. "_gte" for "greater than or equal to".
+
+```jsx
+const postFilters = [
+    <DateInput source="released_gte" label="Released after" />
```
Comment (WiXSL) [inline docs/FilteringTutorial.md:190]:
```suggestion
    <DateInput source="released_gte" label="Released after" />,
```

File: docs/FilteringTutorial.md [hunk 3]
```diff
@@ -171,6 +171,62 @@ If you use the FilterList, you'll probably need a search input. As the FilterLis
 
 ![Filter Live Search](./img/filter-live-search.gif)
 
+## Filter Operators
+
+The internal format for storing filters and sending them to the dataProvider is an object, e.g.:
+
+```js
```
Comment (WiXSL) [inline docs/FilteringTutorial.md:178]:
```suggestion
```json
```

File: docs/FilteringTutorial.md [hunk 4]
```diff
@@ -171,6 +171,62 @@ If you use the FilterList, you'll probably need a search input. As the FilterLis
...
+        const query = {
+            pagination: params.pagination,
+            sort: params.sort,
+            filter: filters,
+        };
+        const url = `${apiUrl}/${resource}?${stringify(query)}`;
+        return httpClient(url).then(({ json }) => ({
+            data: json,
+            total: parseInt(headers.get('content-range').split('/').pop(),10),
+        });
```
Comment (WiXSL) [inline docs/FilteringTutorial.md:224]:
```suggestion
        }));
```

**Review by WiXSL — CHANGES_REQUESTED** [review]

File: docs/FilteringTutorial.md (hunk 3 above)
Comment (fzaninotto) [reply docs/FilteringTutorial.md:178]: no, it's actually JavaScript

**Review by djhi — APPROVED** [review]

**Pull request merged**